utils.js

```diff
@@ -1,11 +1,26 @@
-export const CSS_DOCS_URL = "https://warp-ds.github.io/css-docs"
-export const TECH_DOCS_URL = "https://warp-ds.github.io/tech-docs"
-export const COMING_SOON_MSG = "more info on replacements coming soon"
+export const CSS_DOCS_URL = "https://warp-ds.github.io/css-docs";
+export const TECH_DOCS_URL = "https://warp-ds.github.io/tech-docs";
+
+export const CONSOLE_COLORS = {
```

Comment:
🔥 Great improvement! It is so much readable now!

Reply:
Agree 😄